### DISCUSSION TITLE
Letterboxed mode in Lazer should allow for solid black background

### DISCUSSION THREAD

**Discussion by exylophone** [ppy/osu#27962] [Ideas] [answered]
In letterboxed mode in lazer (i.e. playing on lower resolution than native but still playing in fullscreen), the only options for the background that is accessible are skin, beatmap, and beatmap + storyboard/video, as opposed to stable which is just black. This could be fine if you change the default skin background to black I guess, but for people who have hundreds of skins or regularly download skins, it would just be an unnecessary step for playing letterboxed. I know this is an incredibly niche problem (and I may have just missed a setting?) but still it's worth adding as an option.

**Comment by Joehuu** [comment, marked as answer]
There's already background dim when screen scaling is everything / excluding overlays:

![Screenshot 2024-04-22 at 1 30 16 PM](https://github.com/ppy/osu/assets/35318437/838236b3-6f6c-40ec-b6e8-9c3ef338e51d)
